src/components/ArticlesTable/ArticlesTable.js

```diff
@@ -91,11 +91,11 @@ export class ArticlesTable extends Component {
                     >
                       <MenuItem
                         disabled
-                        primaryText={`Saved: ${moment(item.get('timestamp')).format('lll')}`}
+                        primaryText={`Saved: ${moment.utc(item.get('timestamp')).local().format('lll')}`}
```

Comment:
Well, there are multiple reasons to factor out this single line. Some reasons are important, while others are minor. You may disagree with some, but they sum up to "yes" for me.

The first reason is to give it a name. Naming things makes code easier to read and removes the necessity of the comments. e.g., `format_time_to_user(item.get('timestamp'))` is easier to understand than `moment.utc(item.get('timestamp')).local().format('lll')`.

The second reason is [DRY](https://en.wikipedia.org/wiki/Don't_repeat_yourself) principle. We have two identical lines of code, which the next programmer must remember to change together. They _always_ change together.

There is also [single responsibility principle](https://en.wikipedia.org/wiki/Single_responsibility_principle), which defines that a module should have one, and only one, reason to be changed. (This one is controversial as it depends on how you define responsibilities.)

Anyway, that's not a big issue for me, so I'd rather end the discussion.